meta/group-info/de.yaml

```
@@ -0,0 +1,67 @@
+separator: ", "
+alumni:
+  roles:
+    academy: osu!academy-Gründer
+    admin: Administration
+    chat: Chat Moderator
+    dev: osu!dev
+    media: osu!media Erstellung
+    support: Support-Team
+    tournaments: Verwaltung von Wettbewerben
+gmt:
+  all_mods: alle Moderatoren
+  areas:
+    beatmap: Beatmap-Moderation
+    chat: Chat-Moderation
+    forum: Forum-Moderation
+    team: offizielle Präsenz des Teams
+    dev: osu! Entwicklung
+    player: Spieler-Support
+    support: Technischer Support
+    tournaments: Verwaltung von Wettbewerben
+    wiki: Wiki-Administration
+    mapping: Moderation der Mapping/Modding Community
+nat:
+  areas:
+    communication: Kommunikation
+    evaluation: Beurteilung
+    moderation: Moderation
+    structural: strukturell
+languages:
+  partial: irgendeine <Sprache>
```

```suggestion
  partial: etwas <language>
```
In some articles, I saw that the `some` is used when someone speaks a language only a bit (maybe not a native but fluent enough for communicating). In tables, it would look like this probably:
```md
| ![][flag_PH] [osu user](user id) | some Filipino |
```
So I think `etwas` fits here more.

What I'm not sure about is the _key_ on the right, `<language>`. I don't know if the code implementation requires this to not be changed. Might need an answer from a dev here.

---

i checked the other language files and noticed it has to stay `<language>` but forgot to change back, my bad